src/__tests__/__snapshots__/storeStressSync-test.js.snap

```diff
@@ -115,6 +115,260 @@ exports[`StoreStress (Sync Mode) should handle a stress test for Suspense (Sync
       <Y>
 `;
 
+exports[`StoreStress (Sync Mode) should handle a stress test for Suspense without type change (Sync Mode) 1`] = `
```

Comment:
Just thinking out loud. I'm not asking for you (or anyone) to do this (particularly for fuzz tests) _but_ I found the `Store` snapshot  tests much easier to read when I used named snapshots. Assuming the names are meaningful, this lets me quickly scan the snapshot file to see if anything jumps out as incorrect. That can be hard to do when they're just numbered.

For example:
https://github.com/bvaughn/react-devtools-experimental/blob/131e2e52db486782efe91867611ab9c7021d27e5/src/__tests__/__snapshots__/store-test.js.snap#L28-L76

Reply:
Sorry, I should've clarified that the idea is you shouldn't need to read these snapshots. Most of the tests are _not_ about them.

These snapshots represent the _initial_ steps. But pretty much all failures I've ran into were around updates. In the updates case, we compare the store content to a snapshot, so the snapshot name itself isn't very relevant or useful. What's relevant is the index of the failing case. I'll write up more about the debugging workflow if you'd like later.

I'm not opposed to giving them names. Just saying I haven't had a single "snapshot failure" while debugging this, it's usually the code below that fails. So I'd rather prefer to add more useful logging there. In either case IMO this shouldn't block this PR.

Reply:
> In either case IMO this shouldn't block this PR.

I think my original comment made that super clear. I didn't ask you to change anything, _especially_ for fuzz/stress tests. I was just sharing a thought about snapshots in general.

Reply:
Yeah, that totally makes sense in general.